src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -50,7 +63,7 @@ const ApplicationDetails = ({ onTabSelect, appList, ...props }) => {
                     isFilled
                     className="ins-c-inventory-detail__app-tabs"
                 >
-                    { applications.map((item, key) => (
+                    { activeTabs && activeTabs.length && activeTabs.map((item, key) => (
```

Comment:
This can be simplified with safe accessor
```suggestion
                    { activeTabs?.map((item, key) => (
```